events/message.js

```diff
@@ -67,6 +68,23 @@ module.exports = async (client, message) => {
     }
   }
 
+  //Cooldowns 
```

Comment:
Why was this added?
It's not featured in any production code.

Reply:
This example is from discord.js guide

Reply:
That may be, but the aim of this PR was to update and fix issues is it not?

Reply:
only one addition
should i remove it

Reply:
To be honest, yes. As this feature isn't in any of the other branches/repositories.

Reply:
ok